### PR TITLE
feat: add vector layer to mapbox

### PR DESCRIPTION
**Pull request by markkimsal**

### CODE REVIEW

File: resources/js/Shared/MapboxMap.vue [hunk 1]
```diff
@@ -23,13 +23,55 @@ export default {
...
+                'layout': {},
+                'paint': {
+                'fill-color': '#f08',
+                'fill-opacity': 0.4
+                }
+              },
+              // Insert the layer beneath the first symbol layer.
+              targetLayerId
+          );
+      });
```
Comment (themightychris) [inline resources/js/Shared/MapboxMap.vue:74]: @markkimsal this code is a big blob of red in the project's existing eslint config. I'm opening another PR to tighten up the config

Ensure that your editor is configured to apply the project's eslint configuration while you work, like seen here in vscode:

image

**Review by themightychris — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/TileLayer.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          ->select('tile_data')
+          ->where('zoom_level', $z )
+          ->where('tile_column', $x )
+          ->where('tile_row', $y)
+          ->first();
+
+      if (!$result) {
+          return response('', 404);
+      }
+      $hdrs = [
```
Comment (themightychris) [inline app/Http/Controllers/TileLayer.php:26]: **nitpick**: avoid abbreviations as a general rule on Jarvus projects

**Comment by markkimsal** [comment]
The files that are failing php lint are not in this branch, I can't fix them.

**Comment by themightychris** [comment]
@markkimsal the lint errors all appear to be from the changes merged in #7. Ideally, we should have merged the linting PR first, then rebased and cleaned up that one before merging. Since it's already merged, we'll just need another PR now that re-cleans `main`, and then rebase this one on top again after this merges

**Comment by themightychris** [comment]
@markkimsal I fixed up the formatting of #7  in #8 and then rebased this branch onto the latest `main`, looks good now :+1:

**Pull request merged**